src/components/Buzz/Rewards/DailyBuzzPayout.tsx

```diff
@@ -0,0 +1,282 @@
+import {
+  Center,
+  Grid,
+  Group,
+  Loader,
+  MultiSelect,
+  Paper,
+  ScrollArea,
+  Select,
+  Stack,
+  Text,
+  Title,
+} from '@mantine/core';
+import { Currency, StripeConnectStatus } from '@prisma/client';
+import {
+  BarElement,
+  CategoryScale,
+  Chart as ChartJS,
+  ChartOptions,
+  Tooltip as ChartTooltip,
+  Colors,
+  Legend,
+  LinearScale,
+  PointElement,
+} from 'chart.js';
+import dayjs from 'dayjs';
+import { useMemo, useState } from 'react';
+import { Bar } from 'react-chartjs-2';
+import { useBuzzDashboardStyles } from '~/components/Buzz/buzz.styles';
+import { CurrencyIcon } from '~/components/Currency/CurrencyIcon';
+import { useUserStripeConnect } from '~/components/Stripe/stripe.utils';
+import { getDatesAsList, minDate } from '~/utils/date-helpers';
+import { formatCurrencyForDisplay } from '~/utils/number-helpers';
+import { trpc } from '~/utils/trpc';
+
+ChartJS.register(
+  CategoryScale,
+  LinearScale,
+  PointElement,
+  BarElement,
+  ChartTooltip,
+  Colors,
+  Legend
+);
+
+const now = dayjs();
+const monthsUntilNow = getDatesAsList(now.clone().startOf('year').toDate(), now.toDate(), 'month');
+
+// get date options as month from start of year to now
+const dateOptions = monthsUntilNow.reverse().map((month) => {
+  const date = dayjs(month);
+  return {
+    value: date.toISOString(),
+    label: date.format('MMMM YYYY'),
+  };
+});
+
+export const DailyBuzzPayout = () => {
```

Comment:
I guarantee this name will  get confusing in the future, let's call this component "DailyCreatorCompReward" or something of the sort.